code/style.css

```diff
@@ -1,5 +1,270 @@
+}
+
+.logo {
+  float: right;
+}
+.container2 {
+  position: absolute;
+  top: 0;
+  clear: none;
+  overflow: hidden;
+  background-color: white;
+}
+nav {
+  float: right;
+  position: absolute;
+  top: 0;
+  right: 30%;
+  padding: 30px;
+  overflow: hidden;
+  background-color: white;
+  height: 80px;
+  word-spacing: 5px;
+  margin-left: 50px;
+  margin-bottom: 50px;
+  text-transform: uppercase;
+}
+nav a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  word-spacing: 5px;
+}
+nav a:hover {
+  background-color: #ddd;
+  color: black;
+  border: black;
+  border-radius: 50px;
+}
+
+.appointment-button {
+  background-color: rgba(0, 128, 0, 0.312);
+  color: black;
+  margin: 5px;
+  padding: 10px;
+  border-radius: 50px;
+  word-spacing: 5px;
+  font-weight: 900;
+}
+
+a {
+  color: black;
+  font-style: normal;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.hero-image {
+  background-image: url("./pexels-karolina-grabowska-4239125.jpg");
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  top: 50px;
+  margin-top: 50px;
+  margin-bottom: 10px;
+  width: full-width;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 25%;
+  left: 5%;
+  color: white;
+  font-weight: bold;
+}
+
+button {
+  background-color: rgba(0, 128, 0, 0.312);
+  border-color: rgba(0, 128, 0, 0.312);
+  margin-top: 5px;
+  padding: 10px;
+  border-radius: 5px;
+  word-spacing: 5px;
+  font-weight: bold;
+  color: white;
+}
+button:hover {
+  background-color: gray;
+  border-color: grey;
+  margin-top: 5px;
+  padding: 10px;
+  border-radius: 5px;
+  word-spacing: 5px;
+  font-weight: bold;
+  color: white;
+}
+
+.container {
+  margin-top: 50px;
+  position: relative;
+  width: 33.33%;
+  height: 100%;
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: center;
+  float: left;
+}
+
+.image {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.text-overlay {
+  position: absolute;
+  bottom: 0;
+  background: rgb(0, 0, 0);
+  background: rgba(0, 0, 0, 0.5); /* Black see-through */
+  color: #f1f1f1;
+  width: 100%;
+  transition: 0.5s ease;
+  opacity: 0;
+  color: white;
+  font-size: 20px;
+  padding: 10px;
+  text-align: center;
+}
+
+.container:hover .text-overlay {
+  opacity: 1;
+}
+
+footer {
+  margin-top: auto;
+  height: 100px;
+  color: white;
+  font-weight: bold;
+  width: 100%;
+  background-color: black;
+  min-height: 100px;
+  margin-top: 50px;
+}
+
+.column {
+  height: 100px;
+  width: 33.33%;
+  background-color: black;
+
+  float: left;
+  text-align: center;
+  column-gap: 10px;
+}
+
+.footer-title {
+  margin-top: 20px;
+  text-align: center;
+  font-weight: bolder;
+  font-size: larger;
+  text-transform: uppercase;
+}
+.footer-title1 {
+  text-transform: uppercase;
+}
+.footer-title2 {
+  text-transform: uppercase;
+}
```

Comment:
The cool thing about classes is that you can use them in multiple places in your code. Because they have the same property, you could for example rename it to `footer-title` and use it for both titles in the html (see other comment)